indicators/microsoft-fyfcvk8e.yaml

```diff
@@ -16,7 +19,13 @@ detection:
       dom|contains:
         - THIS WORKS AS A SIGNA TURE CHANGE FOR DETECED BROWSER
 
-    condition: phone and browser
+    licenseServer:
```

Comment:
I believe this modifier is just missing a `|all` on the end, this will ensure that IOK only matches if it finds **all** of these in the requests field.

Reply:
How'd I miss that. Thanks!